tests/integration/test_sqs.py

```diff
@@ -1123,9 +1123,10 @@ def test_post_list_queues_with_auth_in_presigned_url(self):
         # TODO: does not work when testing against AWS
         method = "post"
         protocol = get_service_protocol()
-        port = config.EDGE_PORT_HTTP
+        # CI might not set EDGE_PORT variables properly
```

Comment:
Good catch @baermat . We can set `port = config.EDGE_PORT` here and leave the HTTPS port as `port = 443` below in a future iteration 👍